dev_guide/builds.adoc

```diff
@@ -1426,6 +1432,96 @@ Configuration change triggers currently only work when creating a new
 able to launch a build whenever a `*BuildConfig*` is updated.
 ====
 
+[[build-hooks]]
+
+== Build Hooks
+
+Build hooks can be used as a part of the build flow to run tests inside a
+temporary container and validate your image before it is pushed to a
+registry. The command is executed with the current working directory set to the
+image's `*WORKDIR*` which is your current working directory where your code is
```

Comment:
"which is your current working directory where your code is located"

I think we cannot affirm that for 100% of cases. I'd probably reword this to reflect that it is the common case, but not a guarantee.